core/wire/wire.go

```diff
@@ -0,0 +1,52 @@
+//go:build wireinject
+// +build wireinject
+
+package wire
```

Comment:
I had to put wire into a subfolder, because otherwise `go run ./core/main.go ...` commands don't work (from makefile).